pages/linux/xxhsum.md

```diff
@@ -1,7 +1,7 @@
 # xxhsum
 
 > Print or verify checksums using fast non-cryptographic algorithm xxHash.
-> More information: <https://github.com/Cyan4973/xxHash>.
```

Comment:
Whats with all these changes, unless its changed, we should use links to hubs of information like the githubs, instead of generic man pages.

Reply:
The links provided do not link to comprehensive usage of the commands. Whenever I have to spend more than 30 seconds to look up what options the command supports, I try to replace the link with something that does provide the information immediately. I try to prefer official documentation, but oftentimes official documentation is just a quickstart guide, which overlaps with the purpose of tldr.